api/app/Models/Article.php

```diff
@@ -89,6 +92,21 @@ public static function getValidationRules()
         ];
     }
 
+    protected $mappingProperties = [
```

Comment:
@xiphiaz this is set by the Elasticquent library we are using. I can still change it but I will have the modify ElasticquentTrait, do you want to do this?